staging/istio/templates/configmap-crd.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   name: istio-crd
 data:
   crd.yaml: |-
-{{.Files.Get "files/crd.yaml" | printf "%s" | indent 4}}
+{{.Files.Get "crds/crd.yaml" | printf "%s" | indent 4}}
```

Comment:
Can we somehow integrate these two directories?
```diff
$ diff -Nwrup files/ crds/
diff -Nwrup files/crd.yaml crds/crd.yaml
--- files/crd.yaml      2021-01-06 17:44:18.959912347 +0100
+++ crds/crd.yaml       2021-01-06 17:44:18.959912347 +0100
@@ -2,8 +2,6 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: istiooperators.install.istio.io
-  annotations:
-    "helm.sh/hook": crd-install
   labels:
     chart: istio
 spec:
```

Reply:
Good Catch! I missed that annotation. Let me carry over and test it out.

Reply:
I have included these annotations and verified that helm3 doesn't complain about these extra annotations